Library/ViewModels/CommentRepliesViewModel.swift

```diff
@@ -0,0 +1,53 @@
+import KsApi
+import Prelude
+import ReactiveExtensions
+import ReactiveSwift
+
+public protocol CommentRepliesViewModelInputs {
+  /// Call with the comment that we are viewing replies for. `Comment` can be provided to minimize
+  /// the number of API requests made (ie. no need to find the comment id), but this is for viewing the replies for the root comment.
+  func configureWith(comment: Comment)
+
+  /// Call when the view loads.
+  func viewDidLoad()
+}
+
+public protocol CommentRepliesViewModelOutputs {
+  // TODO: Likely we will be adding `[Comment]` to this output to load replies
```

Comment:
Just flagging this as I'm not sure if you meant to leave this in or not?

Reply:
I did mean to leave it in, but along with @moderateepheezy 's comment above, I updated it to reflect the separation of loading the root comment (only done once) to loading the replies (done every page).